cypress/support/pages/signIn.pageObject.js

```diff
@@ -29,6 +33,10 @@ class SignInPageObject extends PageObject {
     this.signInBtn
       .click();
   }
+
+  checkWrong(text) {
+    this.modalTitle.should('contain.text', text);
```

Comment:
Verify that the `checkWrong` method correctly checks for the error message in the modal title. Ensure that the modal title element is correctly selected and contains the expected text.